lib/Module.js

```diff
@@ -70,7 +70,7 @@ class Module extends DependenciesBlock {
 		this._chunks = new SortableSet(undefined, sortById);
 
 		// Info from Compilation (per Compilation)
-		/** @type {number | string} */
+		/** @type {number} */
```

Comment:
It could also be a string when using `NamedModulesPlugin` or `HashedModuleIdsPlugin`

Reply:
Hmm this was causing some strange assertions, I'll double check here.